src/microbots/bot/LogAnalysisBot.py

```diff
@@ -20,10 +21,11 @@ def __init__(
     ):
         # validate init values before assigning
         bot_type = BotType.LOG_ANALYSIS_BOT
-        permission = PermissionLabels.READ_ONLY
 
-        folder_mount_info = get_path_info(folder_to_mount)
-        base_name = folder_mount_info.base_name
+        folder_mount_info = Mount(
+            folder_to_mount, DOCKER_WORKING_DIR, PermissionLabels.READ_ONLY
+        )
+        base_name = folder_mount_info.host_path_info.base_name
```

Comment:
Do we need this `base_name` member any more?
`folder_mount_info` isn't sufficient?